unit_tests/conftest.py

```diff
@@ -71,8 +107,13 @@ def fixture_docker_scylla(request: pytest.FixtureRequest, params):
     entryfile_path = entryfile_path / 'docker' / 'scylla-sct' / ('entry_ssl.sh' if ssl else 'entry.sh')
 
     alternator_flags = f"--alternator-port {ALTERNATOR_PORT} --alternator-write-isolation=always"
-    docker_version = docker_scylla_args.get(
-        'image', "docker.io/scylladb/scylla-nightly:2025.2.0-dev-0.20250302.0343235aa269")
+
+    # TODO: use vector search capable dev image for vector store tests, until support of vectore store is added to scylla
+    if docker_scylla_args.get('vector_store'):
```

Comment:
I think we can set the image from the test decorator, it would be better then replacing it for of the tests